src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -249,6 +270,29 @@ public Builder schema(@Nullable String schema) {
             return this;
         }
 
+        /**
+         * Configure ssl usage.
+         *
+         * @param ssl
+         * @return this {@link Builder}
+         */
+        public Builder ssl(boolean ssl) {
+            this.ssl = ssl;
+            return this;
+        }
+
+
+        /**
+         * Configure {@link SslProvider} to use.
+         *
+         * @param sslProvider {@link SslProvider} to use
+         * @return this {@link Builder}
+         */
+        public Builder sslProvider(SslProvider sslProvider) {
```

Comment:
We should provide a method that accepts `io.netty.handler.ssl.SslContextBuilder`, too. Providing a `SslContextBuilder` requires fewer configuration objects on the usage side.